x/foundation/msgs_test.go

```diff
@@ -1,6 +1,9 @@
 package foundation_test
 
 import (
+	"fmt"
+	"github.com/line/lbm-sdk/x/auth/legacy/legacytx"
+	"github.com/line/lbm-sdk/x/stakingplus"
```

Comment:
I think it would be better to cover the stakingplus's in its own test. x/stakingplus depends on x/foundation, and the opposite is not true.

Reply:
I found a similar test, so I followed it. 

https://github.com/line/lbm-sdk/blob/1e538735b30a012bafcdb03688a823a421269352/x/foundation/keeper/msg_server_test.go#L9

Reply:
OK. I think it would be better to address this in another PR.